aalondon/static/css/main.css

```diff
@@ -4259,3 +4264,78 @@ body.is-menu-visible #menu {
   }
   
 }
+
+
+/* FOOTER */
+
+#wrapper > footer {
+  margin: 16px;
+}
+
+#footer-container {
+  display: grid;
+  grid-template-columns: 1fr ;
+ 
+  grid-row-gap: 10px;
+  grid-column-gap: 2%;
+  padding-left: 40px;
+  padding-right: 40px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+ 
+  
+  
+}
+
+#footer-links{
+  display:inline-grid;
+  text-transform: uppercase;
+  
+  
+  font-family: "Raleway", Helvetica, sans-serif;
+  margin-top:3%;
+}
+
+
+@media only screen and (min-device-width: 768px)
+{
+  
+  #wrapper > footer {
+    margin-left: 40px;
+    margin-right: 20px;
+  }
+  #footer-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 2fr;
+    grid-template-rows: 1fr 1fr;
+    background-color: white;
+    grid-column-gap: 2%;
+    
+    margin-top:1%;
+    
+    
+    
+  }
+  
+  #footer-links{
+    display:inline-grid;
+    grid-column-start: 1;
+    grid-column-end: 5;
+    grid-template-columns: repeat(6, auto);
+    font-family: "Raleway", Helvetica, sans-serif;
+    margin-top:3%;
+  }
+}
+  
+.side-bar-arrows
+{
+  margin-left: 7%;
+}
+
+#home-reading{
+  margin-bottom: 5%;
+  padding:2%;
+  background: #E6EDFF;
```

Comment:
It's really odd that I don't see this background colour in Firefox

Reply:
This should be fixed with an npm build